coalib/bearlib/aspects/base.py

```diff
@@ -1,8 +1,94 @@
+import re
+
 from coalib.bearlib.languages import Language
 
+import coalib.bearlib.aspects
 from .taste import TasteError
 
 
+class SubaspectGetter:
+    """
+    Special "getter" class to implement get() function in aspectbase that
+    could be accessed from the aspectclass or aspectclass instance.
+    """
+
+    def __get__(self, obj, owner):
+        if obj is not None:
+            parent = obj
+            parent_qualname = type(obj).__qualname__
+        else:
+            parent = owner
+            parent_qualname = owner.__qualname__
+
+        def get_subaspect(subaspect):
+            """
+            Get a subaspect from an aspectclass or aspectclass instance.
+
+            >>> import coalib.bearlib.aspects as aspects
```

Comment:
Let's always `import coalib.bearlib.aspects as coala_aspects` as suggested in https://github.com/coala/coala/issues/4408 . That avoids identifier clashes, since `aspects` is often used as variable, argument, or attribute name